javascript-practice/src/javascripts/constants/messages.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Message for alert the error while fetching or rendering.
+ */
+export const ERROR_MESSAGE = {
+	GET_PPRODUCT_LIST: "Server error! Couldn't load product list",
+	GET_CATEGORY_LIST: "Server error! Couldn't load category list",
+	ADD_CART: "Couldn't add to cart",
+	SERVER_ERROR: "Server Error! Please try again later",
```

Comment:
The message of line 5, 6, 8 look the same? Should we split them like that? Do you have specific case for each error?

Reply:
I fixed it. Please help me check again

Reply:
Update the naming for line 5-6

Reply:
Seems like you missed this message

Reply:
The constant name should show the idea of the message. @mthanh2209